src/apps/companies/apps/edit-one-list/client/EditOneListForm.jsx

```diff
@@ -54,88 +46,79 @@ function ErrorHandler({ errorMessage }) {
             again.
           </StyledH2>
 
-          <StyledP>{errorMessage[API_ERROR]}</StyledP>
+          <StyledP>{message}</StyledP>
         </StatusMessage>
       )}
     </>
   )
 }
 
-function EditOneListForm(state) {
-  const { companyId, companyName, oneListTiers, formInitialValues } = state
-
-  useEffect(() => {
-    if (state[SAVED]) {
-      window.location.href = companies.businessDetails(companyId)
-    }
-  })
-
+function EditOneListForm({
+  companyId,
+  companyName,
+  oneListTiers,
+  formInitialValues,
+}) {
   return (
-    <Task>
-      {(getTask) => (
-        <FormStateful
-          initialValues={formInitialValues}
-          onSubmit={(values) => {
-            getTask(TASK_SAVE_ONE_LIST_DETAILS, STATE_ID).start({
-              payload: {
-                values,
-                companyId,
+    <TaskForm
+      id="edit-one-list"
+      name={TASK_SAVE_ONE_LIST_DETAILS}
+      initialValues={formInitialValues}
+      submissionTaskName={TASK_SAVE_ONE_LIST_DETAILS}
+      analyticsFormName={TASK_SAVE_ONE_LIST_DETAILS}
```

Comment:
```suggestion
      analyticsFormName={editOneListInformation}
```

This needs to be in snake case

Reply:
Thank you!